helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         checksum/config: {{ include (print $.Template.BasePath "/config-repository.yaml") . | sha256sum }}
         checksum/secretDatabase: {{ include (print $.Template.BasePath "/secret-database.yaml") . | sha256sum }}
         checksum/secretS3: {{ include (print $.Template.BasePath "/secret-s3.yaml") . | sha256sum }}
+        checksum/secret-repository: {{ include (print $.Template.BasePath "/secret-repository.yaml") . | sha256sum }}
```

Comment:
Did you tests that? I'm curious to know whether this still works at all.
Also, we don't want to trigger redeploy when some one changes the admin password as it is anyway just used at bootstrap

Reply:
oh that's true, but I added it BEFORE knowing that password is set only at bootstrap :D

more in general, yes it works as expected, and I was thinking to create a new issue to make sure that at least the cm/secret that involves most interesting or most commonly changed values are handled in this way.

Reply:
Agreed. We'll however need to be very cautious about what trigger redeployment as the repo deployment policy is not set to `RollingUpdate` but instead to `Recreate`?
Maybe we should investigate switching to rolling updates first...?